rich/_inspect.py

```diff
@@ -161,11 +155,9 @@ def safe_getattr(attr_name: str) -> Tuple[Any, Any]:
                 yield ""
 
         if self.docs:
-            _doc = getdoc(obj)
+            _doc = self._get_formatted_doc(obj)
             if _doc is not None:
-                if not self.help:
```

Comment:
The process of calling `cleandoc()` on the doctring's content and getting only its first paragraph if `self.help` is not True is now factorised in a single method, rather than done twice in this class